code/__DEFINES/reactions.dm

```diff
@@ -176,15 +176,17 @@
 #define REACTION_OPPRESSION_THRESHOLD 5
 
 // Halon:
-/// The minimum temperature required for halon to form from tritium and BZ.
-#define HALON_FORMATION_MIN_TEMPERATURE 30
-/// The maximum temperature required for halon to form from tritium and BZ.
-#define HALON_FORMATION_MAX_TEMPERATURE 55
-/// The amount of energy 4.25 moles of halon forming from tritium and BZ releases.
-#define HALON_FORMATION_ENERGY 300
+/// Energy released per mole of BZ consumed during halon formation.
+#define HALON_FORMATION_ENERGY 91232.1
```

Comment:
why so specific?

Reply:
It's been like a year since I wrote this, so I don't remember. The most likely case is that the heat capacity of the reaction rises, so I chose an energy release that would converge the temperature to a specific number.

Reply:
oh you're dojn your conservation of energy stuff again, thought that might be the case

Reply:
is it actually reasonable?

Reply:
Not a conservation of energy thing, more of a balancing thing so that the reaction stabilises itself to get to a temperature where it reacts at a reasonable speed, but still has room for optimising if an atmos tech desires.